src/styles/blocks/card.scss

```diff
@@ -0,0 +1,111 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  transition:
+    transform 300ms ease,
+    background-color 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    display: block;
+  }
+
+  &__title {
+    font-family: $font-family;
+    color: #060b35;
+    margin-top: 0;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 300ms ease;
+
+    .card:hover & {
+      color: #34568b;
+    }
+  }
+
+  &__code {
+    font-family: $font-family;
+    color: #616070;
+    margin-top: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__stars {
+    justify-content: space-between;
+    height: 16px;
+    margin-bottom: 24px;
+    display: flex;
+  }
+
+  &__review {
+    font-family: $font-family;
+    color: #060b35;
+    margin: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 16px;
+  }
+
+  &__price {
+    justify-content: space-between;
+    margin-bottom: 16px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__price-tag {
+    font-family: $font-family;
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price-price {
+    font-family: $font-family;
+    color: #060b35;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__link {
+    font-family: $font-family;
+    border: 1px solid $link-hover-color;
+    background-color: $link-hover-color;
+    color: $link-color;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    display: flex;
+    box-sizing: border-box;
+    transition:
+      background-color 300ms ease,
+      color 300ms ease;
+
+    &:hover {
+      color: $link-hover-color;
+      background-color: $link-color;
+    }
+  }
```

Comment:
The `&__link` class is styled as a button. It would be better to make a separate block for a button and make it reusable. Also, the hover effect on the button should change the button's appearance, not the text color. It's more common and user-friendly.